docs/self-hosting/guides/automated-bootstrapping.mdx

```diff
@@ -60,9 +62,95 @@ You can bootstrap an Infisical instance using either the API or the CLI.
     ```
 
     This example command pipes the output through `jq` to extract only the machine identity token, making it easy to capture and use directly in automation scripts or export as an environment variable for tools like Terraform.
+
   </Tab>
 </Tabs>
 
+## Helm Chart Auto Bootstrap
+
+When deploying Infisical using the official Helm chart, you can enable automatic bootstrapping that runs as part of the deployment process. This eliminates the need to manually bootstrap the instance after deployment.
```

Comment:
should we mention that it will also output the token for use by other tools or process?